.run/ConciergeService.run.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+  ~ Copyright (c) 2022 Contributors to the Eclipse Foundation
+  ~
+  ~ See the NOTICE file(s) distributed with this work for additional
+  ~ information regarding copyright ownership.
+  ~
+  ~ This program and the accompanying materials are made available under the
+  ~ terms of the Eclipse Public License 2.0 which is available at
+  ~ http://www.eclipse.org/legal/epl-2.0
+  ~
+  ~ SPDX-License-Identifier: EPL-2.0
+  -->
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="ConciergeService" type="Application" factoryName="Application" folderName="Ditto" nameIsGenerated="true">
+    <option name="ALTERNATIVE_JRE_PATH" value="11" />
```

Comment:
is 11 here correct? We recently upgraded to Java 17. Was this done before the upgrade? In general: is this option required? I just checked my other run configurations and it seems it's not.

Reply:
You're right - that was not added intentionally.

Reply:
I removed this option and it still works.